### PR TITLE
[Compute] Fix Remove-AzVmssVMDataDisk cmdlet and update VM instance view object

### PR DESCRIPTION
**Pull request by hyonholee**
## Description

- Fix Remove-AzVmssVMDataDisk cmdlet
   https://github.com/Azure/azure-powershell/issues/8248

- Update VM instance view object by adding missing properties.

   https://github.com/Azure/azure-powershell/issues/9470

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Compute/Compute/Models/PSVirtualMachineInstanceView.cs [hunk 1]
```diff
@@ -22,6 +22,10 @@ public class PSVirtualMachineInstanceView
         public string ResourceGroupName { get; set; }
 
         public string Name { get; set; }
+        public string ComputerName { get; set; }
```
Comment (markcowl) [inline src/Compute/Compute/Models/PSVirtualMachineInstanceView.cs]: Need a changelog entry for thsi change

Comment (hyonholee) [reply src/Compute/Compute/Models/PSVirtualMachineInstanceView.cs]: @markcowl  Added the changelog.  Thanks.

**Comment by hyonholee** [comment]
@markcowl Any  update?  Could you merge this PR in this release?

**Comment by cormacpayne** [comment]
Closing in favor of https://github.com/Azure/azure-powershell/pull/9713

**Pull request closed without merging**